starfish/plot/intensity_table.py

```diff
@@ -0,0 +1,182 @@
+import warnings
+from typing import Optional
+
+import matplotlib.axes
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+from scipy.stats import pearsonr
+
+from starfish.stats import feature_trace_magnitudes
+from starfish.types import Features
+from .util import annotate_axis
+
+
+def spot_area_histogram(
+        intensities,
+        ax: Optional[matplotlib.axes.Axes],
+        bins: int=25,
+        log=True,
+        **kwargs,
+) -> matplotlib.axes.Axes:
+    """Plot the distribution of spot areas
+
+    Parameters
+    ----------
+    intensities : IntensityTable
+        IntensityTable containing features to be summarized
+    ax : Optional[matplotlib.axes.Axes]
+        Axis to plot in. If not provided, defaults to current axis.
+    bins :
+        Number of bins. Default 25.
+    log : bool
+        Whether to plot the y-axis of the histogram in log scale.
+    kwargs : Dict
+        additional keyword arguments to pass to matplotlib.pyplot.hist
+
+    Returns
+    -------
+    matplotlib.axes.Axes :
+        The axis containing the plot
+
+    """
+    ax = ax if ax is not None else plt.gca()
+    area = (intensities.radius * np.pi) ** 2
+    ax.hist(area, bins=bins, log=log, **kwargs)
+    annotate_axis(
+        ax,
+        title='spot area distribution',
+        ylabel='number of spots',
+        xlabel='area'
+    )
+    return ax
+
+
+def spot_distance_histogram(
+        intensities,
+        ax: Optional[matplotlib.axes.Axes],
+        bins=25,
+        **kwargs,
+) -> matplotlib.axes.Axes:
+    """Plot the distribution of distances of features to the nearest barcode (by metric distance)
+
+    Parameters
+    ----------
+    intensities : IntensityTable
+        IntensityTable containing features to be summarized
+    ax : Optional[matplotlib.axes.Axes]
+        Axis to plot in. If not provided, defaults to current axis.
+    bins :
+        Number of bins. Default 25.
+    kwargs : Dict
+        additional keyword arguments to pass to matplotlib.pyplot.hist
+    intensities
+
+    Returns
+    -------
+    matplotlib.axes.Axes :
+        The axis containing the plot
+
+    """
+    ax = ax if ax is not None else plt.gca()
+    ax.hist(intensities[Features.DISTANCE].values, bins=bins, **kwargs)
```

Comment:
Would be interesting if this were parameterizable so that we could also pass in Features.QUALITY in addition to Features.DISTANCE?